index.html

```diff
@@ -1581,6 +1581,9 @@ <h4><code>ttp:frameRate</code></h4>
           <p>If the document contains any time expression that uses the <code>f</code> metric,
             or any time expression that contains a frames component,
             the <code>ttp:frameRate</code> attribute MUST be present on the <code>&lt;tt&gt;</code> element.</p>
+          <aside class="note">[[TTML2]] requires that the <code>ttp:frameRateMultiplier</code> attribute be used
+           when time expressions relies on a frame rate that is not an integer.
```

Comment:
Would this simplified version work better? If not, need to change "relies" to "rely" or make some other change to fix the grammar.

```suggestion
          <aside class="note">[[TTML2]] specifies the <code>ttp:frameRateMultiplier</code> attribute for
           defining non-integer frame rates.
```